doc/source/whatsnew/v0.20.0.txt

```diff
@@ -75,6 +75,21 @@ Strings passed to ``DataFrame.groupby()`` as the ``by`` parameter may now refere
 
    df.groupby(['second', 'A']).sum()
 
+.. _whatsnew_0200.enhancements.groupby_ngroup:
+
+Groupby Group Numbers
+^^^^^^^^^^^^^^^^^^^^^
+
+A new groupby method ``ngroup``, parallel to the existing ``cumcount``, has been added to return the group order (:issue:`11642`).
```

Comment:
I think the grammar actually works better without the parentheses -- `type(df.groupby("a").ngroup)` is what gives "method", after all.  Should I do this to every mention of `ngroup` in this doc?